src/main/scala/org/broadinstitute/dsde/workbench/sam/directory/JndiDirectoryDAO.scala

```diff
@@ -155,22 +155,13 @@ class JndiDirectoryDAO(protected val directoryConfig: DirectoryConfig)(implicit
     try {
       val identityContext = new BaseDirContext {
         override def getAttributes(name: String): Attributes = {
-          val myAttrs = new BasicAttributes(true)  // Case ignore
+          val myAttrs: BasicAttributes = createPetServiceAccountAttributes(petServiceAccount)
 
           val oc = new BasicAttribute("objectclass")
           Seq("top", "petServiceAccount").foreach(oc.add)
           myAttrs.put(oc)
-
-          myAttrs.put(new BasicAttribute(Attr.email, petServiceAccount.serviceAccount.email.value))
-          myAttrs.put(new BasicAttribute(Attr.sn, petServiceAccount.serviceAccount.subjectId.value))
-          myAttrs.put(new BasicAttribute(Attr.cn, petServiceAccount.serviceAccount.subjectId.value))
-          myAttrs.put(new BasicAttribute(Attr.uid, petServiceAccount.serviceAccount.subjectId.value))
           myAttrs.put(new BasicAttribute(Attr.project, petServiceAccount.id.project.value))
```

Comment:
why wasn't this line pulled out into `createPetServiceAccountAttributes`?

Reply:
because once then entry is created the project can't be changed